src/main/java/duke/GUI/Main.java

```diff
@@ -0,0 +1,32 @@
+package duke.gui;
+
+import java.io.IOException;
+
```

Comment:
Good ordering of import statements, but perhaps line 4 can be removed to keep the formatting consistent?